app/components/CompanyValues.tsx

```diff
@@ -0,0 +1,145 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import { FaRegClock, FaPeopleGroup, FaRegStar } from "react-icons/fa6";
+
+// LOCAL IMPORTS
+import gearBackground from "../../public/images/machine-gear-background.png";
+
+// VALUES COMPONENT DEFINITION
+export default function CompanyValues() {
+  return (
+    <>
+      <Image
+        src={gearBackground}
+        placeholder="blur"
+        fill
+        sizes="100vw"
+        style={{
+          objectFit: "cover",
+          zIndex: -1,
+        }}
+      />
+      <Box
+        bgImage={gearBackground}
+        backgroundRepeat="no-repeat"
+        backgroundSize="cover"
+        p={10}
+        bg="brand.backgroundDark"
+        opacity="90%"
+      >
+        <VStack spacing={8}>
+          <Heading color="white" textAlign="center" mb={4}>
+            {"Kenworthy's Values"}
+          </Heading>
+          <VStack
+            maxW="1400px"
+            bg="white"
+            borderRadius="2xl"
+            px={32}
+            py={24}
+            spacing={36}
+          >
+            <HStack spacing={12} align="center">
+              <VStack align="start" spacing={4}>
+                <Heading size="lg">Quality</Heading>
+                <Box
+                  bgGradient="linear(90deg, #DB0A40 0%, rgba(219, 10, 64, 0.04) 96%, rgba(219, 10, 64, 0) 100%)"
+                  w="full"
+                  h={1}
+                ></Box>
+                <Text fontWeight="bold" fontSize="lg">
+                  {
+                    "At Kenworthy Machine, producing quality parts to our customer's specifications is our most important job."
+                  }
+                </Text>
+                <Text fontSize="lg">
+                  {
+                    "Our team is dedicated to making sure your parts are correct an in tolerance every time. They use their expertise, as well as modern inspection equipment to achieve this goal. Upon request we can provide inspection reports for your parts with detailed measurements compared to drawing specifications. We also offer laser engraving to give your parts a professional appearance."
+                  }
+                </Text>
+              </VStack>
+              <Spacer />
+              <Flex
+                w={28}
+                h={28}
+                bg="brand.backgroundDark"
+                borderRadius="2xl"
+                align="center"
+              >
+                <Icon as={FaRegStar} w={24} h={24} color="white" />
+              </Flex>
```

Comment:
This `Flex` container holds the icon, and is `Flex` vs `Box` so that the `flex` properties of `align` and `justify` can be used. As you can see on line 73, the width is set, but if you inspect the element in dev tools, you'll notice the width shrinks. I'm not sure how to rearrange the `HStack` containing these elements to give enough room for this container. Adjusting the `spacing` property in the outermost `HStack` (line 52) does help make room, but then the text block and icon aren't spaced out enough. Ideally the text container gets narrower while the space and icon's proportions stay the same.

Reply:
See my answer above :)

Reply:
Also, for this icon you use size 24 for width and height, but for the other two icons it's size 28. Not sure if it's intentional.

Reply:
Oh yes I forgot to comment on that, I was trying to show that when the icon gets smaller the proportions of the outer box around the icon get more rectangle-like vs square, but the icon looks better smaller. I believe your above suggestions help with this